lib/utils.ts

```diff
@@ -57,13 +53,23 @@ export class Utils {
 
     let preWeb3;
 
-    if (typeof web3 !== "undefined") {
+    let globalWeb3;
+
+    if ((typeof global !== "undefined") && (global as any).web3) {
```

Comment:
Why not just define web3 in the global namespace as you used to?

Reply:
I want to undefine `web3` as global because so the TypeScript compiler will enforce the use of Utils.getWeb3().  `web` is not always accessible globally.